message/message.go

```diff
@@ -77,6 +80,49 @@ type messageRequest struct {
 	ScheduledDatetime string      `json:"scheduledDatetime,omitempty"`
 }
 
+// path represents the path to the Message resource.
+const path = "messages"
+
+// Read retrieves the information of an existing Message.
+func Read(c *messagebird.Client, id string) (*Message, error) {
+	message := &Message{}
+	if err := c.Request(message, http.MethodGet, path+"/"+id, nil); err != nil {
+		return nil, err
+	}
+
+	return message, nil
+}
+
+// List retrieves all messages of the user represented as a MessageList object.
+func List(c *messagebird.Client, msgListParams *MessageListParams) (*MessageList, error) {
+	messageList := &MessageList{}
+	params, err := paramsForMessageList(msgListParams)
+	if err != nil {
+		return messageList, err
+	}
+
+	if err := c.Request(messageList, http.MethodGet, path+"?"+params.Encode(), nil); err != nil {
+		return nil, err
+	}
+
+	return messageList, nil
+}
+
+// Create creates a new message for one or more recipients.
+func Create(c *messagebird.Client, originator string, recipients []string, body string, msgParams *MessageParams) (*Message, error) {
```

Comment:
What do you think about calling this function `Send`?

Reply:
Hmm, within the wider context of resources, I'd say `Create` is a bit more suitable - I can definitely understand where you're coming from for message/mmsmessage, but it'd feel odd to have `message.Send()` but `lookup.Create()`. What's your opinion on this?